### PR TITLE
JBPM-6841: Review details page layout. Details alignment, input field…

### PR DESCRIPTION
**Pull request by nmirasch**
…s size, primary button review and task context tab removed

### CODE REVIEW

**Comment by nmirasch** [comment]
@cristianonicolai @bsremac some of the full screen details revision added. Could you take a look? thanks!

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskdetails/TaskDetailsPresenter.java [hunk 1]
```diff
@@ -98,6 +100,14 @@ protected void setTaskDetails(String status,
         view.setUser(actualOwner);
         view.setTaskStatus(status);
         view.setTaskPriority(priority);
+        if (processInstanceId == null) {
+            view.setProcessInstanceId("None");
```
Comment (cristianonicolai) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskdetails/TaskDetailsPresenter.java]: @nmirasch perhaps hide these fields in the UI if null?

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskcomments/TaskCommentsViewImpl.html [hunk 2]
```diff
@@ -3,23 +3,22 @@
         <div class="form-horizontal">
             <fieldset>
                 <div class="form-group">
-                    <label class="control-label col-md-3" for="newTaskCommentTextArea" data-field="newTaskCommentLabel"></label>
-                    <div class="col-md-9">
+                    <label class="control-label col-md-1" for="newTaskCommentTextArea" data-field="newTaskCommentLabel"></label>
```
Comment (cristianonicolai) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskcomments/TaskCommentsViewImpl.html]: perhaps use _col-md-2_ to align with other tabs spacing?

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskcomments/TaskCommentsViewImpl.html [hunk 3]
```diff
@@ -3,23 +3,22 @@
...
+                    <div class="col-md-5">
                         <input type="text" data-field="newTaskCommentTextArea" id="newTaskCommentTextArea" placeholder="" class="form-control"/>
                     </div>
                 </div>
                 <div class="form-group">
-                    <div class="col-md-offset-3 col-md-9">
+                    <div class="col-md-offset-1 col-md-9">
```
Comment (cristianonicolai) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskcomments/TaskCommentsViewImpl.html]: same here _col-md-offset-2_

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskcomments/TaskCommentsViewImpl.html [hunk 4]
```diff
@@ -3,23 +3,22 @@
...
                     </div>
                 </div>
             </fieldset>
         </div>
         <div data-field="listContainer">
-            <div id="commentsListGrid" data-field="commentsListGrid" />
-            <!--<table id="commentsListGrid" data-field="commentsListGrid" class="table table-bordered table-striped table-hover">-->
-            <!--</table>-->
-            <!--<div id="pager" class="pagination pagination-right pull-right" data-field="pager"></div>-->
+            <div class="col-md-11">
```
Comment (cristianonicolai) [inline jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskcomments/TaskCommentsViewImpl.html]: _col-md-12_ to use full width

**Comment by cristianonicolai** [comment]
@nmirasch Task Admin tab is missing some restyling as well, update button to primary and adjust form content to left, review user input size. Is that something you will do in another PR?

**Comment by nmirasch** [comment]
@cristianonicolai @bsremac Added the remaining changes about task Admin, applied the same alignment to task comments, make invisible the task process related fields. Removed the fixed height of the process diagram.

File: jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskdetails/TaskDetailsPresenter.java (hunk 1 above)
Comment (bsremac) [reply jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskdetails/TaskDetailsPresenter.java]: @nmirasch @cristianonicolai I wonder if this this is the right place to even consider tasks without Porcess Instance ID or Process ID. Taking into account that starting from v7 there can be no tasks outside the process context, if those tasks even appear they apparently represent an illegal state and a bug on Kie Server side.
I think this validation should be done somewhere in backend code instead. Not sure what are the options there, though, as if we are just taking the result set from Kie Server and sending it directly to the task list without any validation, this is then not possible.
What I'm trying to say is that this task shouldn't even be listed to begin with, hardly to get to the point of looking at its details in UI.

**Review by cristianonicolai — APPROVED** [review]

**Comment by bsremac** [comment]
@nmirasch In Process Details tab, _Sub Processes_ and _Services_ values are not aligned horizontally
screenshot from 2018-02-28 22-23-33

**Pull request merged**